tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -162,6 +162,27 @@ module.exports = {
       for (const btn of cancelButtons) {
         await this.api.elementIdClick(btn.ELEMENT).waitForAnimationToFinish()
       }
+    },
+    browseToUserProfile: function () {
```

Comment:
Rename this function. It's kind of confusing as we use these kinds of name when we directly browse using the url.

Reply:
No, we haven't used similar function names for browsing directly using the url..So I think this function name is suitable here